### PR TITLE
Results 1/N (total 2/N)

### PR DESCRIPTION
**Pull request by jharenza**
❗ Merge #157 first ❗ 

### Purpose

add partial results for fig 1 

### Issue

_What GitHub issue does your pull request address?_
NA

#### Which areas should receive a particularly close look?
- website citation for manubot - not sure if this will make a real citation

#### Is there anything that you want to discuss further?
- added stub for CCDL process
- removed old figure - don't think we have this in panel yet to add new link

#### Is the pull request ready for review?
yes

### Pull review checklist

Unless otherwise noted above, this PR will be considered ready for review when all four items have been checked.

- [x] All changes to text follow "one sentence per line" [[Manubot instructions](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/USAGE.md#manubot-markdown)]
- [x] All citations follow the [Manubot citation instructions](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/USAGE.md#citations)
- [x] All changes or additions to tables follow the [Manubot table instructions](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/USAGE.md#tables)
- [x] All figures follow the [Manubot figure instructions](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/USAGE.md#figures)
- [x] There are no new spelling errors identified by the [Manubot spellchecker](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/USAGE.md#spellchecking)

### Spellcheck Step

The dictionary used for spellchecking can be updated.
Edit the file in `build/assets/custom-dictionary.txt` by adding new entries to the end.
You do not need to change anything else.
However, if you want to update the first line to have an accurate count of words and you want to remove non-unique ones, run the following command from within `build/assets` on your favorite OS X or Linux machine:
```
(( len = $(awk '!a[$0]++' < custom-dictionary.txt | wc -l ) - 1 )); tmpfile="$(mktemp)"; echo "personal_ws-1.1 en $len utf-8" > $tmpfile; tail -n +2 custom-dictionary.txt | awk '!a[$0]++' >> $tmpfile; mv $tmpfile custom-dictionary.txt
```

### CODE REVIEW

**Comment by jaclyn-taroni** [comment]
@jharenza since your branch is in this repository and not on your branch, you can in fact stagger these pull requests so the diff only shows the changes for this PR and **not** #157. You can do that by [changing the base branch](https://docs.github.com/en/github/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/changing-the-base-branch-of-a-pull-request).  In this example, we'd change the base branch for this pull request to be `intro`. I will go ahead and make that change, but FYI - can make things go a bit more smoothly.

File: content/04.results.md [hunk 1]
```diff
@@ -2,13 +2,15 @@
...
+We then harnessed the benchmarking efforts of the KidsFirst Data Resource Center to develop a robust and reproducible data analysis workflow within the CAVATICA platform to perform primary somatic analyses (variant calling of single nucleotide variants (SNVs), copy number variants (CNVs), structural variants (SVs), and fusions, *Figure S1* - red boxes and *STAR Methods*). 
+Next, we created a Github analysis repository (https://github.com/AlexsLemonade/OpenPBTA-analysis) with CircleCI (Continuous Integration) to ensure analysis reproducibility and a GitHub manuscript repository (https://github.com/AlexsLemonade/OpenPBTA-manuscript) with ManuBot (@https://manubot.org/) integration to enable manuscript creation using Markdown within GitHub. 
+We maintained a data release folder on Amazon S3 containing merged files for each analysis, downloadable from the GitHub repository (https://github.com/AlexsLemonade/OpenPBTA-analysis/#data-access-via-download-script) or open access CAVATICA project (https://cavatica.sbgenomics.com/u/cavatica/openpbta/).
```
Comment (jaclyn-taroni) [inline content/04.results.md:10]: I don't think these URLs need to be here necessarily. It strikes me as potentially confusing. I expect that the CAVATICA project will be in data availability?

File: content/04.results.md [hunk 2]
```diff
@@ -2,13 +2,15 @@
 
 Results section stub.
 
-### The Open Pediatric Brain Tumor Atlas
+### Crowd-sourced Somatic Analysis to create an Open Pediatric Brain Tumor Atlas
```
Comment (jaclyn-taroni) [inline content/04.results.md:5]: I suspect this header will be more fitting once we fill in the stub.

File: content/04.results.md [hunk 3]
```diff
@@ -2,13 +2,15 @@
...
 
-This section will introduce the dataset (e.g., the histologies represented and what data types are included; Figure {@fig:openpbta-overview}A-B) and the process for contributing analytical code and to the manuscript (Figure {@fig:openpbta-overview}C-D).
+We previously performed WGS, WXS, and RNA-Seq on matched tumor and normal tissues as well as selected cell lines from 943 patient tumors from the Pediatric Brain Tumor Atlas (PBTA) (@doi:10.1093/neuonc/noz192), samples sourced from the Children’s Brain Tumor Network (CBTN) and the PNOC003 DMG clinical trial (@doi:10.1002/ijc.32258) of the Pacific Pediatric Neuro-oncology Symposium (PNOC) (*Figure 1*).
```
Comment (jaclyn-taroni) [inline content/04.results.md:7]:
```suggestion
We previously performed WGS, WXS, and RNA-Seq on matched tumor and normal tissues as well as selected cell lines from 943 patient tumors from the Pediatric Brain Tumor Atlas (PBTA) [@doi:10.1093/neuonc/noz192], samples sourced from the Children’s Brain Tumor Network (CBTN) and the PNOC003 DMG clinical trial [@doi:10.1002/ijc.32258] of the Pacific Pediatric Neuro-oncology Symposium (PNOC) (*Figure 1*). 
```

**Comment by jharenza** [comment]
> changing the base branch

Oh, that is good to know, thanks :)

File: content/04.results.md (hunk 1 above)
Comment (jharenza) [reply content/04.results.md:10]: yeah you're right - they can go into data availability (worked on this first but should go there)

**Pull request merged**